code/modules/mining/abandonedcrates.dm

```diff
@@ -41,11 +41,9 @@
 		if(21 to 25)
 			for(var/i in 1 to 5)
 				new /obj/item/weapon/contraband/poster(src)
-		if(26 to 30)
+		if(26 to 35)
 			for(var/i in 1 to 3)
 				new /obj/item/weapon/reagent_containers/glass/beaker/noreact(src)
-		if(31 to 35)
-			new /obj/item/seeds/cash(src)
```

Comment:
Yeah, I intentionally let this die; I know @Aurorablade expressed interest in the budget version of cargo, and this would horrendously wreck it.